spec/unit/appmgmt_spec.rb

```diff
@@ -46,7 +46,7 @@ def compile_to_env_catalog(string)
 
       application app {
         prod { one: host => ahost, export => Cap[cap] }
-        cons { two: host => ahost, consume => Cap[cap] }
```

Comment:
A missing parameter doesn't work. It will yield and error. So there's no such thing as testing that "a missing parameter works". This test verifies that consume works, period. It tests that consume results in that a parameter that otherwise would be missing isn't missing.